src/commonMain/kotlin/kotools/types/text/NotBlankString.kt

```diff
@@ -15,13 +15,15 @@ import kotools.types.experimental.ExperimentalTextApi
 import kotools.types.number.StrictlyPositiveInt
 import kotools.types.number.toStrictlyPositiveInt
 import kotlin.jvm.JvmInline
+import kotlin.jvm.JvmName
 
 /**
  * Returns this string as an encapsulated [NotBlankString], or returns an
  * encapsulated [IllegalArgumentException] if this string is
  * [blank][String.isBlank].
  */
 @SinceKotoolsTypes("4.0")
+@JvmName("of")
 public fun String.toNotBlankString(): Result<NotBlankString> =
     NotBlankString of this
```

Comment:
This function could include an example in its documentation for calling it from Java code.